exp-player/addon/components/exp-player.js

```diff
@@ -4,12 +4,13 @@ import parseExperiment from '../utils/parse-experiment';
 
 export default Ember.Component.extend({
     layout: layout,
+    store: Ember.inject.service('store'),
```

Comment:
See comments below: as ember stands at present, it's not a great idea to make components own data or interact with the store directly. (hence the refactor moving that responsibility into the route, and saving via an action) 

Once I've had a chance to review the rest of the PR we can discuss concrete ways to work the new code into that scheme.